### PR TITLE
fix(nms): Enable type checking for organizations and fix issues

### PR DESCRIPTION
**Pull request by thmsschmitt**
## Summary

The organizations were part of `fbc_js_core` for which type checking was disabled. When integrating `fbc_js_core` we noticed that there were many type errors in organizations so we temporarily disabled the type checker.

This PR re-enables the type checking and fixes the issues.
   
## Test Plan
Checked that creating, deleting and editing an organizations still works.

### CODE REVIEW

File: nms/app/views/organizations/OrganizationEdit.js [hunk 1]
```diff
@@ -264,7 +274,7 @@ function OrganizationEdit(props: WithAlert & Props) {
               await axios.delete(
                 `/host/organization/async/${organization?.id || ''}`,
               );
-              history.push('/host/organizations');
+              navigate('/host/organizations');
```
Comment (thmsschmitt) [inline nms/app/views/organizations/OrganizationEdit.js:277]: This broke the back button

File: nms/app/views/organizations/OrganizationEdit.js [hunk 2]
```diff
@@ -275,7 +285,7 @@ function OrganizationEdit(props: WithAlert & Props) {
               <Grid container alignItems="center">
                 <Grid>
                   <IconButton
-                    onClick={() => history.goBack()}
+                    onClick={() => navigate(-1)}
```
Comment (thmsschmitt) [inline nms/app/views/organizations/OrganizationEdit.js:284]: This broke the back button

File: nms/app/views/organizations/OrganizationEdit.js [hunk 3]
```diff
@@ -264,7 +274,7 @@ function OrganizationEdit(props: WithAlert & Props) {
               await axios.delete(
                 `/host/organization/async/${organization?.id || ''}`,
               );
-              history.push('/host/organizations');
```
Comment (thmsschmitt) [inline nms/app/views/organizations/OrganizationEdit.js:263]: history was undefined here

File: nms/app/views/organizations/OrganizationUsersTable.js [hunk 4]
```diff
@@ -59,7 +59,7 @@ function OrganizationUsersTable(props: OrganizationUsersTableProps) {
               props.tableRef.current?.onQueryChange();
             })
             .catch(() => {
-              enqueueSnackbar(`Unable to delete user: ${user.name}`, {
+              enqueueSnackbar(`Unable to delete user: ${user.id}`, {
```
Comment (sebathomas) [inline nms/app/views/organizations/OrganizationUsersTable.js:62]: Why change the name to ID here?

Comment (thmsschmitt) [reply nms/app/views/organizations/OrganizationUsersTable.js:62]: There is no `name` on user

**Review by Neudrino — APPROVED** [review]
Looks reasonable to me.

**Review by sebathomas — APPROVED** [review]

**Review by Neudrino — APPROVED** [review]

**Pull request merged**